plugins/performance-lab/includes/site-health/audit-autoloaded-options/helper.php

```diff
@@ -147,7 +147,10 @@ function perflab_aao_query_autoloaded_options(): array {
 	$large_options = array();
 
 	foreach ( $all_options as $option_name => $option_value ) {
-		if ( strlen( $option_value ) > $option_threshold ) {
+		if ( is_array( $option_value ) || is_object( $option_value ) ) {
+			$option_value = serialize( $option_value ); // phpcs:ignore WordPress.PHP.DiscouragedPHPFunctions.serialize_serialize
+		}
+		if ( strlen( $serialized_value ) > $option_threshold ) {
```

Comment:
Oops. I didn't update this variable:

```suggestion
		if ( strlen( $option_value ) > $option_threshold ) {
```